### PR TITLE
Do not replace any TXT-records

### PR DESCRIPTION
**Pull request by jwulf0**
Do not replace any TXT-records but rather POST new ones in order to comply with the challenge.

Running the example from your README (with one certificate for both the example-domain and a wildcard certificate below it) kept failing for me. It seems that the challenge expects multiple TXT-records at the same time with the same record name (when running the certbot in manual mode for, there is an explicit hint that this is in compliance with the DNS standard) but the plugin replaces the one it had just created.

This should fix it. Unfortunately, I don't know a lot about python, so I didn't adjust any unit tests. Feel free to merge, modify or ignore this... :)

### CODE REVIEW

File: certbot_dns_ionos/dns_ionos.py [hunk 1]
```diff
@@ -114,22 +114,21 @@ def _api_request(self, type, action, data = None):
...
         elif type == 'delete':
             resp = requests.delete(url, headers=self.headers)
         else:
             raise errors.PluginError(
                 "HTTP Error during request. Unknown type {0}".format(type)
             )
-        logger.debug("API REquest to URL: %s", url)
-        if resp.status_code != 200:
+        logger.debug("API Request to URL: %s", url)
+        if not resp.ok:
```
Comment (jwulf0) [inline certbot_dns_ionos/dns_ionos.py:128]: The newly introduced POST requets is responded to with `201` by the current IONOS API appearantly.

File: certbot_dns_ionos/dns_ionos.py [hunk 2]
```diff
@@ -114,22 +114,21 @@ def _api_request(self, type, action, data = None):
...
             content = json.loads(resp.content)
-            error_msg = resp.reason + " " + content['message']
             raise errors.PluginError(
-                "HTTP Error during request {0}:{1}".format(resp.status_code, error_msg)
+                "HTTP Error during request {0}:{1}".format(resp.status_code, content)
```
Comment (jwulf0) [inline certbot_dns_ionos/dns_ionos.py:131]: It seems, this message-key isn't guaranteed to be there on the newly introduced POST-request so I made this a bit more generic.

**Comment by jwulf0** [comment]
And only now I realize, in the `dev` something like this has already been merged :D well, this might make this obsolete, idk...

**Comment by helgeerbe** [comment]
Hi @jwulf0 I'm sorry, that you you had this issue. I totally forgot to merge the changes from dev to main. Thanks for reminding me.  I will prepare a new release today. This little plugin runs on my home server and does not need any care.

**Pull request closed without merging**